source/puppet/5.4/config_ssl_external_ca.markdown

```diff
@@ -102,16 +102,19 @@ Agent SSL certificate private key | `puppet config print hostprivkey --section a
 Root CA certificate               | `puppet config print localcacert --section agent`
 Root certificate revocation list  | `puppet config print hostcrl --section agent`
 
-## Option 2: Puppet server functioning as an intermediate CA
+## Option 2: Puppet Server functioning as an intermediate CA
 
-Puppet Server can operate as an intermediate CA to an external root CA.  (The server cannot be an intermediate to an intermediate.)  In this mode, the Puppet CA is left enabled; it can automatically accept CSRs and distribute certificates to agents, and can use the standard `puppet cert` command to sign certificates. However, there are some limitations:
+Puppet Server can operate as an intermediate CA to an external root CA. (The server cannot be an intermediate to an intermediate.)  In this mode, the Puppet CA is left enabled; it can automatically accept CSRs and distribute certificates to agents, and can use the standard `puppet cert` command to sign certificates. However, there are some limitations:
 
 * Agent-side CRL checking is not possible, although Puppet Server still verifies the CRL.
-* The CA certificate bundle (the external root CA combined with the intermediate CA certificate) must be distributed to the agents manually, ideally before puppet runs
+* The CA certificate bundle (the external root CA combined with the intermediate CA certificate) must be distributed to the agents manually, ideally before puppet runs.
+* Puppet Server does not support a passphrase-protected CA key.
```

Comment:
nit, in this context `CA key` is slightly ambiguous. Could you clarify that it's the intermediate CA private key (since that's the one the clojure CA needs to load to sign/issue certs)?